matrix_sdk_base/src/event_emitter/mod.rs

```diff
@@ -174,111 +176,113 @@ mod test {
     pub struct EvEmitterTest(Arc<Mutex<Vec<String>>>);
 
     #[async_trait::async_trait]
+    // we don't need to test our tests right?
+    #[cfg_attr(tarpaulin, skip)]
```

Comment:
Just wanted to make sure this didn't get lost. This is ok to skip right?

Reply:
I think Tarpaulin does that automatically or will do so in the future, so we might skip adding unnecessary `cfg_attr` lines, they are harmless since they will be counted as covered. It only skews our percentage a bit but that's ok